src/Providers/FlightsProvider/FlightsProvider.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {FC, ReactNode, useEffect, useState } from "react";
+import axios from "axios";
+import { EmptyFlight, Flight } from "typings/flights.d";
+
+interface IFlightContext {
+  flights: Flight[],
+  currentFlight: Flight,
+  switchCurrentFlight: (id: number) => void,
+  newest: Flight
+}
+
+export const FlightsContext = React.createContext<IFlightContext>({
+  flights: [],
+  currentFlight: EmptyFlight,
+  switchCurrentFlight: () => {},
+  newest: EmptyFlight,
+});
```

Comment:
```suggestion
export const FlightsContext = React.createContext<IFlightContext>({} as IFlightContext);
```

Not sure if this isn't against good practices, but in this way you don't have to provided 'empty state' for context in this place.